src/Windows/Avalonia.Win32/Win32TypeExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using static Avalonia.Win32.Interop.UnmanagedMethods;
+
+namespace Avalonia.Win32
+{
+    public static class Win32TypeExtensions
```

Comment:
Probably worth making this whole class internal rather than having a public class with a single internal method?

Reply:
done